integration/fuse/bin/alluxio-fuse

```diff
@@ -118,7 +119,19 @@ fuse_stat() {
   fi
 }
 
-USAGE_MSG="Usage:\n\talluxio-fuse [mount|umount|stat]"
+USAGE_MSG="Usage:\n\talluxio-fuse [mount|umount|stat]
+
+mount \tmount an Alluxio path to local file system
+umount \tdismount an Alluxio path from local file system
```

Comment:
"dismount" is not really a good  word to use in this context. Just say `unmount`

Reply:
done